EventView-1/src/main/java/com/eventview/batch/ScheduleConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.eventview.batch;
+
+import com.eventview.service.EmailService;
+import com.eventview.service.EventService;
+import com.eventview.service.EventTypeService;
+import com.eventview.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ScheduleConfig {
+
+    @Autowired
+    private EventService eventService;
+    @Autowired
```

Comment:
only event service shd return all required info no need of all service calls here

Reply:
got it will fix